### ISSUE TITLE
[BUG] evpn.transit_vni: True not rewritten in some cases

### ISSUE THREAD

**Issue by jbemmel** [ipspace/netlab#827]
### Describe the bug

The evpn.transit_vni: True attribute is not replaced by an actual VNI when referencing the vrf from a group. As a result, the generated topology.yml for leaf1 contains:
```
vrfs:
  tenant:
    evpn:
      evi: 1
      rd: 10.0.0.1:1
      transit_vni: true  <-- wrong
```

I suspect that the attribute gets copied before being rewritten by the evpn module.

### To Reproduce

netlab create <topology.yml>

### Expected behavior

A topology with the correct transit VNI value assigned

### Lab topology

```
defaults.vxlan.start_vni: 20000

message: |
  This test case illustrates an issue with the order of parameter copying/evaluation

defaults.provider: clab
defaults.device: srlinux

groups:
  leafs:
    members: [ leaf1 ]
    module: [ vlan,bgp,evpn,vrf ]
    vrfs: # Commenting this out fixes things
     tenant:
    vlans:
      red:

vrfs:
 tenant:
  evpn.transit_vni: True # FRR does not support asymmetrical

vlans:
 red:
  vrf: tenant

bgp.as: 65000
evpn.vlans: [ red ]

nodes:
  leaf1:
```

### Output

```
[snippet]
vrfs:
  tenant:
    evpn:
      evi: 1
      rd: 10.0.0.1:1
      transit_vni: true
```

### Version
netlab version 1.5.4

**Comment by ipspace** [comment]
Ah, so the bug only happens when you define a VRF within the group **and** globally? Every time we stumble across something like this I say to myself "I know it was a bad idea to implement so many ways to configure stuff, I should have said NO" ;) Will fix.

**Comment by ipspace** [comment]
FWIW, it's not group-related. The bug is triggered by "copy global VRF data into node VRF data if the node VRF exists" code. It's reproducible with:

```
defaults.device: eos
module: [ vlan, bgp, evpn, vrf ]

vrfs:
  tenant:
    evpn.transit_vni: True

vlans:
  red:
    vrf: tenant

bgp.as: 65000
nodes:
  a:
    vrfs:
      tenant:
    vlans:
      red:
```

**Comment by jbemmel** [comment]
Then the next question is if also
```
nodes:
 a:
  vrfs:
   tenant:
    evpn.transit_vni: True
```
works as expected

**Comment by ipspace** [comment]
That one throws an error saying "you can't use that on a node VRF definition" (which makes sense)

**Comment by ipspace** [comment]
Fixed in #830
